infra/terraform/terraform.go

```diff
@@ -38,9 +38,11 @@ func New(stateDir string, config Config) (*terraform, error) {
 			constants.FieldProvisioner: "terraform",
 			constants.FieldCluster:     config.ClusterName,
 			"state-dir":                stateDir,
+			"plugin-dir":               config.PluginDir,
 		}),
-		Config:   config,
-		stateDir: stateDir,
+		Config:    config,
+		stateDir:  stateDir,
+		pluginDir: config.PluginDir,
```

Comment:
Since we're already embedding `Config` into the `terraform` struct, it is not necessary to duplicate the `pluginDir`.

Reply:
Excellent point.  I'm still getting a hang of the struct embedding in go.

Reply:
Addressed in 0abc4e7.